metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -1610,3 +1613,97 @@ def visit_join_conversion_events_node(self, node: JoinConversionEventsNode) -> S
                 from_source_alias=output_data_set_alias,
             ),
         )
+
+    def visit_window_reaggregation_node(self, node: WindowReaggregationNode) -> SqlDataSet:  # noqa: D102
+        from_data_set = node.parent_node.accept(self)
+        parent_instance_set = from_data_set.instance_set  # remove order by col
+        parent_data_set_alias = self._next_unique_table_alias()
+
+        metric_instance = None
+        order_by_instance = None
+        partition_by_instances: Tuple[MdoInstance, ...] = ()
+        for instance in parent_instance_set.as_tuple:
+            if instance.spec == node.metric_spec:
+                metric_instance = instance
+                continue
+            if instance.spec == node.order_by_spec:
+                order_by_instance = instance
+            if instance.spec in node.partition_by_specs:
+                partition_by_instances += (instance,)
```

Comment:
These are mutually exclusive, right? The metric can't be a partition or order by, and the node doesn't allow the order by to be a partition spec.

Should we make this if/elif/elif?

Reply:
Good catch. I wrote this originally thinking that the order by could also be in the partition by, and then refactored later but missed this!